viewing_party/main.py

```diff
@@ -0,0 +1,184 @@
+# WAVE 1 *****************************************************************
+
+# create a dictionary from data title, genre, rating
+
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+# add watched movie to watched list
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+# add movie to list of movies to watchlist
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data 
+
+# remove movie from watchlist and move to watched 
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title: # if title is found in watchlist and watched list
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+    return user_data 
+
+# WAVE 2 *****************************************************************
+
+# get average rating of movie
+def get_watched_avg_rating(user_data):
+    sum = 0
+    avg_rating = 0.0
+
+    if len(user_data["watched"]) == 0:
+        return avg_rating
+
+    for movie in user_data["watched"]:
+        sum = sum + movie["rating"]
+
+    avg_rating = sum / len(user_data["watched"])
+    return avg_rating
```

Comment:
let's see if we can get this to work with just one return statement. either way we will return avg_raiting right? so let's try this:

```suggestion
    sum = 0
    avg_rating = 0.0

    if len(user_data["watched"]) > 0:
        for movie in user_data["watched"]:
            sum = sum + movie["rating"]

        avg_rating = sum / len(user_data["watched"])
    return avg_rating
```